src/settings/toml/manifest.rs

```diff
@@ -512,6 +516,96 @@ impl Manifest {
     }
 }
 
+#[derive(Debug, Clone, Default, PartialEq)]
+pub struct LazyAccountId(OnceCell<String>);
+
+impl Serialize for LazyAccountId {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: serde::Serializer,
+    {
+        self.0.get().serialize(serializer)
+    }
+}
+
+impl<'de> Deserialize<'de> for LazyAccountId {
+    fn deserialize<D>(deserializer: D) -> Result<Self, D::Error>
+    where
+        D: serde::Deserializer<'de>,
+    {
+        Option::deserialize(deserializer).map(Into::into)
+    }
+}
+
+impl From<Option<String>> for LazyAccountId {
+    fn from(opt: Option<String>) -> Self {
+        let cell = OnceCell::new();
+        if let Some(val) = opt {
+            cell.set(val).unwrap();
+        }
+        Self(cell)
+    }
+}
+
+impl LazyAccountId {
+    /// Return the `account_id` in `wrangler.toml`, if present.
+    ///
+    /// Use this with caution; prefer `maybe_load` instead where possible.
+    fn if_present(&self) -> Option<&String> {
+        self.0.get()
+    }
+
+    /// If `account_id` can be inferred automatically, do so;
+    /// otherwise, return `None`.
+    ///
+    /// Note that *unlike* `load`, this will never prompt the user or warn.
+    pub(crate) fn maybe_load(&self) -> Option<String> {
+        use crate::commands::whoami::fetch_accounts;
+        use crate::settings::global_user::GlobalUser;
```

Comment:
Do you mind pulling these use statements (along with the others in various places) up to the top of the file, so it's consistent with the rest of the code? it also is nice to avoid duplication.